mockgcp/mockcloudfunctions/v1.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mockcloudfunctions
+
+import (
+	"context"
+
+	pb "google.golang.org/genproto/googleapis/cloud/functions/v1"
```

Comment:
We need to migrate away from the deprecated proto?

Reply:
Now that we've merged #863 we shouldn't need to